src/lasp_core.erl

```diff
@@ -556,45 +556,23 @@ read_var(Id, Threshold0, Store, Self, ReplyFun, BlockingFun) ->
 %% @doc Perform a read (or monotonic read) for a series of particular
 %%      identifiers.
 %%
-%%      @todo track in dag
 %%
 -spec read_any([{id(), value()}], pid(), store()) ->
     {ok, var()} | {ok, not_available_yet}.
 read_any(Reads, Self, Store) ->
```

Comment:
I assume this change is ok, I looked into the implementation of `read_any` in the [plumtree module](https://github.com/lasp-lang/lasp/blob/unstable/src/lasp_plumtree_broadcast_distribution_backend.erl#L364) and it was already using the `read` function